CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,43 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * Get Financial Account type relations
+   *
+   * @param $flip bool
```

Comment:
I want the function to return
1. class::getfinancialAccountRelations(); 
  array(
1 => 4,
2 => 3,
3 => 4,
4 => 1,
5 => 1,
6 => 2,
);
key = Account relationship value = financial_account_type_id
2. class::getfinancialAccountRelations(TRUE); 
array(
1 => array(4, 5),
2 => array(6),
3 => array(2),
4 => array(1,3),
);
value = array of Account relationship key = financial_account_type_id

but array_flip() flips key/value pairs which i don't want to be in  my case since financial_account_type_id can have more than 1 account_relationship associated with it. 

Should i change the parameter name to avoid confusion?